File: Java/BookStoreOOP/src/model/BillDetail.java
package model;

public class BillDetail {
    private int quantity;
    private Bill bill;
    private Object object;

    public BillDetail(String id, int quantity, Bill bill, Object object) {
        this.quantity = quantity;
        this.bill = bill;
        this.object = object;
    }

    public int getQuantity() {
        return quantity;
    }

    public Bill getBill() {
        return bill;
    }

    public Object getObject() {
        return object;
    }

    public void setQuantity(int quantity) {
        this.quantity = quantity;
    }

    public void setBill(Bill bill) {
        this.bill = bill;
    }

    public void setObject(Object object) {
        this.object = object;
    }
}

File: C#/BookStoreOOP/BookStoreOOP/Model/BillDetail.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace BookStoreOOP.Model
{
    public class BillDetail
    {
        private int quantity;
        private Bill bill;
        private Book book;

        public BillDetail(int quantity, Bill bill, Book book)
        {
            this.quantity = quantity;
            this.bill = bill;
            this.book = book;
        }

        public int getQuantity()
        {
            return quantity;
        }

        public Bill getBill()
        {
            return bill;
        }

        public Book getBook()
        {
            return book;
        }

        public void setQuantity(int quantity)
        {
            this.quantity = quantity;
        }

        public void setBill(Bill bill)
        {
            this.bill = bill;
        }

        public void setBook(Book book)
        {
            this.book = book;
        }
    }
}
